frontend/src/modules/browse/map-landing.jsx

```diff
@@ -21,38 +28,64 @@ const MapLanding = ({
   setToggleButton,
   updateQuery,
 }) => {
-  const { innerWidth, innerHeight } = window;
-  const { profile, countries, landing, nav } = UIStore.useState((s) => ({
+  const {
+    profile,
+    countries,
+    landing,
+    nav,
+    transnationalOptions,
+  } = UIStore.useState((s) => ({
     profile: s.profile,
     countries: s.countries,
     landing: s.landing,
     nav: s.nav,
+    transnationalOptions: s.transnationalOptions,
   }));
   const [country, setCountry] = useState(null);
   const [counts, setCounts] = useState("project");
+  const [multiCountry, setMultiCountry] = useState(null);
+  const [multiCountryCountries, setMultiCountryCountries] = useState(null);
 
   const isApprovedUser = profile?.reviewStatus === "APPROVED";
   const hasProfile = profile?.reviewStatus;
 
-  const isLoaded = () => !isEmpty(countries) && !isEmpty(landing?.map);
+  const isLoaded = () =>
+    !isEmpty(countries) &&
+    !isEmpty(landing?.map) &&
+    !isEmpty(transnationalOptions);
 
   const clickCountry = (name) => {
     setToggleButton("list");
     updateQuery("country", name);
     history.push(`/browse?country=${name}`);
   };
 
+  const handleChangeTab = (key) => {
+    key === "multi-country" && setCountry(null);
```

Comment:
should these lines better expressed as a conditional?

Reply:
I mean simple conditional
```js
if(){
} else if(){
}
```

Reply:
Hmm, yep,
I will change it to a  simple conditional